tests/Functional/DebugModeTest.php

```diff
@@ -102,7 +103,7 @@ public function testDebugModeInline(): void
         $zipArchive = new ZipArchive();
         $zipArchive->open($fileName);
         $config = $zipArchive->getFromName('config.json');
-        $config = \GuzzleHttp\json_decode((string) $config, true);
+        $config = json_decode((string) $config, true);
```

Comment:
Nevim jaky je rozdil mezi standardnim `json_encode` a tim z Guzzle, ale tady vidim krasnou nachtavku, kdy si na prvni dobrou pomyslis, ze je to standardni `json_encode` :smile:

Reply:
to je dobrá poznámka! používali jsme ten guzzlovej, protože je type safe, ale v testech je to celkem k prdu